internal/services/oracle/oracle_autonomous_database_clone_from_database_data_source.go

```diff
@@ -0,0 +1,570 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package oracle
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-03-01/autonomousdatabases"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/oracle/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+var _ sdk.DataSource = AutonomousDatabaseCloneFromDatabaseDataSource{}
+
+type AutonomousDatabaseCloneFromDatabaseDataSource struct{}
+
+type AutonomousDatabaseCloneFromDatabaseDataSourceModel struct {
+	Name              string            `tfschema:"name"`
+	ResourceGroupName string            `tfschema:"resource_group_name"`
+	Location          string            `tfschema:"location"`
+	Tags              map[string]string `tfschema:"tags"`
+
+	// Clone-specific properties
+	SourceAutonomousDatabaseId string `tfschema:"source_autonomous_database_id"`
+	CloneType                  string `tfschema:"clone_type"`
+	ReconnectCloneEnabled      bool   `tfschema:"reconnect_clone_enabled"`
+	RefreshableClone           bool   `tfschema:"refreshable_clone"`
+	RefreshableModel           string `tfschema:"refreshable_model"`
+	RefreshableStatus          string `tfschema:"refreshable_status"`
+	TimeUntilReconnectUtc      string `tfschema:"time_until_reconnect_utc"`
```

Comment:
please follow this [naming guidance](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md)
```suggestion
	TimeUntilReconnectUtc      string `tfschema:"time_until_reconnect_in_utc"`
```